README.md

```diff
@@ -34,8 +34,10 @@ pip3 install -U xformers --index-url https://download.pytorch.org/whl/cu118
 pip3 install -U xformers --index-url https://download.pytorch.org/whl/cu121
 # [linux & win] cuda 12.4 version
 pip3 install -U xformers --index-url https://download.pytorch.org/whl/cu124
-# [linux only] (EXPERIMENTAL) rocm 6.1 version
-pip3 install -U xformers --index-url https://download.pytorch.org/whl/rocm6.1
+# [linux & win] cuda 12.6 version
+pip3 install -U xformers --index-url https://download.pytorch.org/whl/cu126
```

Comment:
I believe 12.1 is going to be dropped, can you remove it?

Reply:
yeah, it's true